src/main/scala/projectEuler/LargestPrimeFactor.scala

```diff
@@ -0,0 +1,26 @@
+package projectEuler
+
+import scala.math._
+
+/**
+ * Problem 3. Largest prime factor
+ * The prime factors of 13195 are 5, 7, 13 and 29.
+ * What is the largest prime factor of the number 600851475143 ?
+ * @see https://projecteuler.net/problem=3
+ */
+class LargestPrimeFactor(num:Long) {
```

Comment:
I would recommend using a less ambiguous name, probably `LargestPrimeFactorCalculator` to show that this class is not the largest prime factor, it is the class that *computes* the largest factor